lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -94,10 +93,6 @@ public Stemmer(Dictionary dictionary) {
       word = scratchBuffer;
     }
 
-    if (dictionary.isForbiddenWord(word, length)) {
```

Comment:
Hunspell happily stems forbidden words, too, and so do we

Reply:
Hmmm... shouldn't this be reported as a bug to hunspell though?

Reply:
From my correspondence with Hunspell's author it seems to be a feature, that stemming gives some results even for misspelled words